### PR TITLE
UDX-162-guides-example-selector-updates

### PR DESCRIPTION
**Pull request by jaffrepaul**
https://cypress-io.atlassian.net/browse/UDX-162

Additional pass at updating selectors in code examples to align with our best practices. Specifically, https://github.com/cypress-io/cypress-documentation/issues/4227.

### NOTE: this effort is only to become consistent with our best practices. A follow up convo will take place on revising our best practices and these will be easier to find and replace once they're consistent.

### CODE REVIEW

File: content/guides/guides/debugging.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ execution and add a `debugger` at the appropriate time:
 ```js
 cy.visit('/my/page/path')
 
-cy.get('.selector-in-question').then(($selectedElement) => {
+cy.get('[data-testid=selector-in-question]').then(($selectedElement) => {
```
Comment (admah) [inline content/guides/guides/debugging.md:69]:
```suggestion
cy.get('[data-testid="selector-in-question"]').then(($selectedElement) => {
```

I think the attribute values need quotes around them like [this example](https://docs.cypress.io/api/commands/get#Find-5-elements-with-the-given-data-attribute).

**Review by admah — CHANGES_REQUESTED** [review]
Correct me if I'm wrong, but I think all of these types of selectors:

[data-testid=completed]

should have quotes around the attribute values like:

[data-testid="completed"]

File: content/guides/guides/debugging.md (hunk 1 above)
Comment (jaffrepaul) [reply content/guides/guides/debugging.md:69]: Technically, I think that's more correct. I used the example that we show at the bottom of the table here as a guide https://docs.cypress.io/guides/references/best-practices#How-It-Works and confirmed this works. 

I was going for consistency with these edits but totally open to making a larger scale find and replace if we decide we want something else.

Comment (jaffrepaul) [reply content/guides/guides/debugging.md:69]: Or...maybe we should update that table to be more technically correct and the examples follow that.

Comment (admah) [reply content/guides/guides/debugging.md:69]: @jaffrepaul my vote would be to update the table to avoid similar questions in the future.

Comment (jaffrepaul) [reply content/guides/guides/debugging.md:69]: I can dig that. I'll need to make another pass and update some other selectors. Easy find and replace tho.

**Review by admah — APPROVED** [review]
LGTM 🚀

**Pull request merged**